### ISSUE TITLE
\hbar should not be bold

### ISSUE THREAD

**Issue by kevinbarabash** [mathjax/MathJax#1674]
<img width="329" alt="screen shot 2016-12-04 at 12 15 29 am" src="https://cloud.githubusercontent.com/assets/1044413/20864248/caed5904-b9b6-11e6-9274-5ab9ab7ab791.png">

**Comment by kevinbarabash** [comment]
I'm confused as to why this is happening.  In https://github.com/mathjax/MathJax-dev/blob/master/fonts/OTF/TeX/makeFF#L1146-L1156 it shows that the glyph for `\hbar` comes from `msbm10` which appears to be the right font if http://www.math.union.edu/~dpvc/jsmath/download/extra-fonts/msbm10/msbm10.html can be trusted.  If the correct glyph can't be located maybe we can construct it the same way that bold `\hbar` is on https://github.com/mathjax/MathJax-dev/blob/master/fonts/OTF/TeX/makeFF#L2089-L2093.

**Comment by dpvc** [comment]
Yes, it should be the correct glyph, but it does seem to be bolder than it should be.  Perhaps it has to do with the mftrace setup for the msbm10 font that is causing a difference in weight.  There is a bolder version available in the bold-italic font, so this one clearly is intended as the non-bold version.

While a technique for constructing the hbar like the one you cite would work, you would need to make it a bit more sophisticated in that it would have to take the initial "h" from the italic font, not the regular font (in which the hbar is being created).

In the meantime, you could use use the equivalent of

    \def\hbar{\mathit{\rlap{\unicode{xAF}}h}}

As an aside, note that `\mathbf` has (and should have) no effect on `\hbar`, so your `\mathbf{\hbar}` should get the same glyph as `\hbar`.

**Comment by dpvc** [comment]
Fixed in v4.0.
